### ISSUE TITLE
Documentation on non-AMICI-based objective functions

### ISSUE THREAD

**Issue by paulstapor** [ICB-DCM/pyPESTO#478]
After a short chat at COMBINE conference, I think it would be very helpful to underline (either in the docs or somewhere else) that pyPESTO doesn't have to use AMICI, but is basically agnostic about the objective function structure.
One could add an example, which is a bit more complex than the Rosenbrock function. That might help users, who want to use pyPESTO e.g. with another simulator.
What do you think?

**Comment by dweindl** [comment]
May be good to highlight more, but I don't think it's too helpful to add another example. However simple, the Rosenbrock example shows how to provide a custom objective function. Or what specifically would you want to show in addition?

**Comment by FFroehlich** [comment]
I think renaming (and maybe refactoring) existing notebooks according to what they are intended to demonstrate would go a long way. Agree with @dweindl that adding more isn't necessarily helpful.

**Comment by paulstapor** [comment]
> I think renaming (and maybe refactoring) existing notebooks according to what they are intended to demonstrate would go a long way. Agree with @dweindl that adding more isn't necessarily helpful.

Yes, fully agreed. Basically, Rosenbrock should show this somehow. However, it's very, very simple (e.g., always computes objective + gradient + Hessian). So, maybe something which illustrates this capability in more detail might be helpful. Then, no additional notebook is needed.
